Source/Core/DolphinWX/Config/InterfaceConfigPane.cpp

```diff
@@ -162,7 +162,7 @@ void InterfaceConfigPane::LoadGUIValues()
 
 void InterfaceConfigPane::LoadThemes()
 {
-	auto sv = DoFileSearch({"*"}, {
```

Comment:
If you look at the new DoFileSearch, you'll see it treats an empty vector the same as a vector with just an empty string in it, so there's no need to do that.